src/test/java/com/sfuentes/CarLicensePlateTest.java

```diff
@@ -0,0 +1,109 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CarLicensePlateTest {
+
+  @Test
+  void getCountOfEachColorByLicensePlate_FiveLicensePlate_OneOfEachColor() {
```

Comment:
you are repeating "FiveLicensePlate" for each test name, to me seems that it adds no value to the reader, maybe you can just remove it and the tests names will still be good enough.

Reply:
fixed